libraries/log4j.rb

```diff
@@ -15,19 +15,19 @@ def render_appender(name, options)
         when :layout
           content += render_layout(prefix, value)
         else
-          if value.respond_to?(:call)
-            content << %(#{prefix}.#{camelcase(key)}=#{value.call})
-          else
-            content << %(#{prefix}.#{camelcase(key)}=#{value})
-          end
+          content << if value.respond_to?(:call)
+                       %(#{prefix}.#{camelcase(key)}=#{value.call})
+                     else
+                       %(#{prefix}.#{camelcase(key)}=#{value})
+                     end
```

Comment:
Personally, I think that the changed version is more difficult to read, and it hurts my eyes :)